python/tskit/genotypes.py

```diff
@@ -33,25 +33,24 @@
 class Variant:
     """
     A variant in a tree sequence, describing the observed genetic variation
-    among samples for a given site. A variant consists of (a) the **alleles** that
-    may be observed at the sample nodes for this site; (b) the **genotypes**
-    mapping sample IDs to the observed alleles and (c) of a reference to
-    the :class:`Site` that the Variant has been decoded at.
+    among samples for a given site. A variant consists of (a) a tuple of
+    **alleles** listing the potential allelic states which samples at this site
+    can possess; (b) the **genotypes** mapping sample IDs to the observed alleles
+    and (c) a reference to the :class:`Site` at which the Variant has been decoded.
```

Comment:
This should also say that a list of samples is included.

Reply:
Good point. Done